docs/rules/rename-default-import.md

```diff
@@ -0,0 +1,56 @@
+# import/rename-default-import
+
+This rule will enforce an alias for a default package import. Only ES6 imports are processed.
```

Comment:
i don't think "an alias" is the proper terminology here; import names are called "bindings", not aliases.

Reply:
However, when you take a look at the https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import the docs refer to those parts as `alias{number}`